packages/core/src/global/global.scss

```
@@ -33,7 +33,7 @@
   --ion-color-warning-shade: var(--color-contextual-warning-dark-1);
   --ion-color-warning-tint: var(--color-contextual-warning-light-1);
 
-  --ion-color-danger: var(--color-contextual-danger-regular);
+  --ion-color-danger: var(--color-contextual-error-light-1);
   --ion-color-danger-rgb: var(--color-contextual-danger-regular-rgb);
```

We need to change `rgb` too

```suggestion
  --ion-color-danger-rgb: var(--color-contextual-error-light-1-rgb);
```